datarepo-clienttests/src/main/java/testscripts/ScalePodsUpDown.java

```diff
@@ -0,0 +1,79 @@
+package testscripts;
```

Comment:
Note: This is the existing test, but most of the code has been factored out into BulkLoadUtils.